src/clang-cpp-frontend/clang_cpp_convert.h

```diff
@@ -16,14 +16,46 @@ class clang_cpp_convertert : public clang_c_convertert
   virtual ~clang_cpp_convertert() = default;
 
 protected:
+  typedef std::unordered_map<std::size_t, std::pair<std::string, typet>>
+    this_mapt;
+  this_mapt this_map;
+
   bool get_decl(const clang::Decl &decl, exprt &new_expr) override;
 
+  // Get reference to declared variabales or functions, e.g:
```

Comment:
Can I ask you to follow the standard for comments as in: https://github.com/esbmc/esbmc/blob/master/src/goto-symex/goto_symex_state.h?